Mathlib/CategoryTheory/Preadditive/HomOrthogonal.lean

```diff
@@ -93,7 +93,12 @@ noncomputable def matrixDecomposition (o : HomOrthogonal s) {α β : Type} [Fint
     biproduct.matrix fun j k =>
       if h : f j = g k then z (f j) ⟨k, by simp [h]⟩ ⟨j, by simp⟩ ≫ eqToHom (by simp [h]) else 0
   left_inv z := by
-    ext (j k)
+    -- Porting note: `ext j k` used to achieve the next four steps,
```

Comment:
This is not true, as I mentioned [elsewhere](https://github.com/leanprover-community/mathlib4/pull/4340#discussion_r1210974051); lean 3 applied `hom_ext'` first and the proof below does not reflect the original lean 3 proof.